packages/types/src/session.ts

```diff
@@ -128,6 +130,8 @@ export interface SessionResource extends ClerkResource {
   attemptSecondFactorVerification: (
     params: SessionVerifyAttemptSecondFactorParams,
   ) => Promise<SessionVerificationResource>;
+  attemptFirstFactorPasskeyVerification: (nonce: string | null) => Promise<SessionVerificationResource>;
+  attemptSecondFactorPasskeyVerification: (nonce: string | null) => Promise<SessionVerificationResource>;
```

Comment:
❓ As i can see, we don't expose dedicated strategy methods. Should we keep it as is and remove those as well?

Reply:
Yeah we don't have for the other strategies but in order to make passkeys easier in custom flows some additional logic is encapsulated there to ease the use, this is how we do it for `SingIn` as well